recipes/qt-msbuild/all/conanfile.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from conan import ConanFile
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.53.0"
```

Comment:
I haven't tested if the recipe works with Conan 1.x, I only tested with the latest 2.x version.

I've seen that other similar recipes also do
```python
self.env_info.QtMsBuild = os.path.join(self.package_folder, "res")
```
which seems to be for backwards-compatibility with Conan 1.x. Should I add that or should I rather specify
```python
required_conan_version = ">=2.0"
```
?